docs/sources/setup/upgrade/_index.md

```diff
@@ -36,13 +36,35 @@ The output is incredibly verbose as it shows the entire internal config struct u
 
 ## Main / Unreleased
 
+## 3.0.0
+
+Loki 3.0 is a major version increase and comes with several breaking changes.
+
+Here is the shortlist of things we think most people may encounter:
+
+  * Structured metadata is enabled by default and requires `tsdb` and `v13` schema or Loki won't start. See [Structured Metadata, Open Telemetry, Schemas and Indexes](#structured-metadata-open-telemetry-schemas-and-indexes)
+  * `shared_store` config is removed. See [Removed `shared_store` and `shared_store_key_prefix` from shipper configuration](#removed-sharedstore-and-sharedstorekeyprefix-from-shipper-configuration)
+  * Loki now enforces a max line size of 256KB by default (you can disable this or increase this but this is how we run Loki). See [Changes to default configure values](#changes-to-default-configuration-values-in-30)
+  * Loki now enforces a max label limit of 15 labels per series, down from 30. Extra labels inflate the size of the index and reduce performance, you should almost never need more than 15 labels. See [Changes to default configure values](#changes-to-default-configuration-values-in-30)
+  * Be aware of a lot of metric name changes See [Distributor metric changes](#distributor-metric-changes), [Embedded cache metric changes](#embedded-cache-metric-changes), and [Metrics namespace](#metrics-namespace)
+
+If you would like to see if your existing config will work with Loki 3.0:
+  * In an empty directory on your computer, put your config into a file named `loki-config.yaml`
+  * Run this command from that directory `docker run --rm -t -v "${PWD}":/config grafana/loki:3.0.0 -config.file=/config/loki-config.yaml -verify-config=true`
+  * Note: if you introduce a new schema_config entry it may cause additional validation errors
```

Comment:
```suggestion
{{< admonition type="note" >}}
If you introduce a new `schema_config` entry it may cause additional validation errors.
{{< /admonition >}}
```